mapping/leaflet.md

```diff
@@ -0,0 +1,32 @@
+# Leaflet
+[Leaflet](https://leafletjs.com/) is an open-source JavaScript library for highly interactive, mobile-friendly maps.
+
+There are two ways to use Leaflet:
+1. [Leaflet](https://leafletjs.com/) for plain JavaScript projects.
+2. [react-leaflet](https://react-leaflet.js.org/), which is for React projects. [See our documentation](/mapping/react-leaflet.md) for some guidance on how to use it.
+
+## Leaflet with vanilla JS
+The Leaflet library is a tool to use when:
+1. you need a ["slippy map"](https://wiki.openstreetmap.org/wiki/Slippy_Map)
+2. your map has a lot of interactions, more than you can manage with an SVG
+3. you have some GeoJSON or custom geographical features
+
+[The Leaflet documentation](https://leafletjs.com/) is the best place to learn the basics of Leaflet. 
+
+### Examples from DataMade projects
+- [Searchable Map Template](https://github.com/datamade/searchable-map-template-csv)
+- [Justice Divided](https://github.com/datamade/justice-divided/blob/master/js/district_map.js)
+- [Cal FWD CDI site](https://github.com/datamade/california-dream-index/blob/master/cdi/static/js/detailMap.js)
+- [Councilmatic](https://github.com/datamade/django-councilmatic/blob/5b074f376667766e4b6dbf093871d294bb35fc51/councilmatic_core/templates/councilmatic_core/council_members.html#L105)
+
+
+### Examples from Leaflet's documentation
+- [Quickstart](https://leafletjs.com/examples/quick-start/)
+- [Using GeoJSON with Leaflet](https://leafletjs.com/examples/geojson/)
+- [Interactive Choropleth Map](https://leafletjs.com/examples/choropleth/)
+ 
+### Limitations
+tk
```

Comment:
would be nice to have some limitations here. i'm not knowledgeable enough to offer any insight